src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+
+- Influence degrades reasonably quickly if placed in the Sec cells and exposed to calming jazz music.
+
+### Influence ticks up when exposed to sources and slowly ticks down when not. There is a max Influence level.
+
+Addition of NT and Revolutionary posters and propaganda, that both sides, and traitors, can put up and take down across the station.
+
+These posters can be printed across the station by both sides and gives Security Cadets, Lawyers and bored Passengers something to do, even on non-Rev rounds, and a reason for Sec to patrol (imagine!). They’re an DOT effect with a max cap per item, and add or subtract Influence from those who see them.
+
+Putting up pro-Revolution posters is added as a Traitor goal to make them have a reason to appear outside Revs.
+
+Clowns get the ability to put up annoying "Visit Clown Planet" tourism posters.
+
+Posters can be be destroyed by spraying them with paint or tearing them down.
+
+### Addition of Loyalty Loudspeakers in Sec
+
+These Loyalty Loudspeakers are audio emitters that reduce a player’s Influence level, even if they’re converted. If a player who is converted de-converts, their Influence immediately drops to zero. This is a new machine that requires quite a bit of power to run. A button is added to the Warden’s office to disable and enable all loudspeakers. The loudspeakers play calming, smooth jazz.
+
+These Loudspeakers cannot be made on the station. One of the key jobs of Sec is to ensure Sec remains safe; if it's been bombed and the Loudspeakers are inoperable, the Revolution is succeeding.
+
+### Head Revs can use an implant ability to silently convert characters with high enough Influence to being a Rev.
+
+This replaces flashing outright, and makes conversion a stealthy game. Revs do not know who the Head Revs are unless the Head Rev outs themselves.
+
+### Add the Interrogator, an electric chair that tests Influence levels
+
+The Interrogator is a high-power-draw device that lives in Sec. This machine does three things:
+
+1. Minorly fries the person placed inside it. `Your hair stands on end!` If you're a Diona, you are set on fire. `The lightning ignites your limbs!` It does no damage to Slimes. `You feel tingly`.
+2. Reads their Influence level.
+3. Causes the lights to flicker station-wide.
+
+This allows Sec to ensure anyone they're releasing from prison has a low influence level.
+
+It can be emagged to dramatically increase the power draw, EMPing the station for a period of time, and instantly frying any poor sod put inside it.
+
+### Add the Centcomm Uplink Terminal as an indestructable core console on the Bridge.
+
+This terminal is the core security uplink between the space station and the rest of the universe (in fiction). It can send announcements and notifies who is the current chain-of-command.
+
+This console is indestructable, protected by military-grade force fields, immovable and powered by a nuclear engine that will outlast the star it's anchored to.
```

Comment:
Is this not the existing Communications Console that's in the bridge and cap's office?

Reply:
There's multiple comms consoles but the one on the bridge needs to be upgraded to be the "special" one that is the logical centre of the entire space station, as it's the flag that the Revs want to capture